addons/web/static/src/xml/base.xml

```diff
@@ -780,6 +780,11 @@
         t-att-class="disabled ? 'oe_list_button_disabled' : ''"
         ><img t-attf-src="#{prefix}/web/static/src/img/icons/#{widget.icon}.png"
         t-att-alt="widget.string"/></button>
+<button t-name="ListView.row.buttonwidget" type="button"
+        t-att-title="widget.string" t-att-disabled="disabled || undefined"
+        t-att-class="disabled ? 'oe_list_button_disabled' : '',invisible ? 'oe_form_invisible' : ''"
+        ><img t-attf-src="#{prefix}/web/static/src/img/icons/#{widget.icon}.png"
+        t-att-alt="widget.string"/></button>
```

Comment:
If you need a template for a widget in account_asset, add it in some xml file in addons/account_asset, not in web client